Lib/test/test_email/test__header_value_parser.py

```diff
@@ -1700,6 +1700,12 @@ def test_get_display_name_ending_with_obsolete(self):
         self.assertEqual(display_name[3].comments, ['with trailing comment'])
         self.assertEqual(display_name.display_name, 'simple phrase.')
 
+    def test_get_display_name_bpo_32178(self):
```

Comment:
I don't think we generally add a suffix in the test case name referring to the issue. Is there a reason for adding this in?

Reply:
There is really no hard and fast rule for or against it IIUC, but I couldn't come up with a better name to differentiate this test case :)

Reply:
Well, I guess we can name this: 

```suggestion
    def test_get_display_name_for_invalid_address_fields(self):
```

Reply:
What do you think about this?

Reply:
Thanks, I updated the name and added a comment to link to bpo instead.